src/lesson4/HomeWork.java

```diff
@@ -0,0 +1,15 @@
+package lesson4;
+
+public class HomeWork {
+    public static void main(String[] args) {
+        String name = "Pavlo Moskovchuk";
+
+        System.out.println(name.toUpperCase());
+        System.out.println(name.toLowerCase());
+        System.out.println();
+
+        int l = name.length();
```

Comment:
make name not 'l' but 'length'. because 'l' doesn't mean nothing

Reply:
managed to work with project again.  package back and edited.